### ISSUE TITLE
`REGEXP_LIKE(NULL, 'pattern')` give `FALSE` in PostgreSQL, but `NULL` in Oracle

### ISSUE THREAD

**Issue by ukolovda** [orafce/orafce#146]
The expression
`oracle.REGEXP_LIKE(null, '\d*')`
returns `false` in PostgreSQL

But in Oracle it returns `NULL`:
```sql
begin
  if REGEXP_LIKE(null, '\d*') is null then
    raise_application_error(-20000, 'IS NULL!');
  else
    raise_application_error(-20001, 'NOT IS NULL!');
  end if;
end;
```

Result is different in a condition:
`IF NOT REGEXP_LIKE(pVALUE, '\d*')`
when `pVALUE = NULL`

**Comment by ukolovda** [comment]
> @ukolovda, please, canyou check this fix [695a908](https://github.com/orafce/orafce/commit/695a908cb2effbe0b2fa0242dee234279604840c) ?

It works fine, thank you!

**Comment by okbob** [comment]
ne 31. 10. 2021 v 15:46 odesílatel Dmitry Ukolov ***@***.***>
napsal:

> @ukolovda <https://github.com/ukolovda>, please, canyou check this fix
> 695a908
> <https://github.com/orafce/orafce/commit/695a908cb2effbe0b2fa0242dee234279604840c>
> ?
>
> It works fine, thank you!
>

super

Thank you for bug report

Pavel
